src/apis/sourceimages.js

```diff
@@ -215,15 +221,17 @@ export default (state) => {
    * ```
    *
    * @authenticated
-   * @param  {string} organization    name
-   * @param  {string} url             The URL to the remote image
-   * @param  {Object} [metadata=null] optional, metadata to be added, either user or dynamic
+   * @param  {string} organization     name
+   * @param  {string} url              The URL to the remote image
+   * @param  {Object} [metadata=null]  optional, metadata to be added, either user or dynamic
+   * @param  {{optimize_source: bool}} [options={}] Optional: only {optimize_source: true/false} yet, false is default
    * @return {Promise}
    */
-  sourceimages.createByUrl = (organization, url, metadata = null) => {
-    const options = {
+  sourceimages.createByUrl = (organization, url, metadata = null, options = {}) => {
+    const config = {
       form: true
     }
```

Comment:
you could also try:
```
const config = {
  form: true,
  ...options
}
```

Reply:
ah, wrongly read the code in question. let's leave it as it is.

Reply:
Was at a different place, but added that now